dex/networks/eth/params.go

```diff
@@ -70,6 +74,22 @@ var v0Gases = &Gases{
 	RefundGas: 43000,
 }
 
+// LoadGenesisFile loads a Genesis config from a json file.
+func LoadGenesisFile(genesisFile string) (*core.Genesis, error) {
+	fid, err := os.Open(genesisFile)
+	if err != nil {
+		return nil, err
+	}
+	defer fid.Close()
+
+	var genesis core.Genesis
+	err = json.NewDecoder(fid).Decode(&genesis)
+	if err != nil {
+		return nil, fmt.Errorf("unable to unmarshal simnet genesis: %v", err)
+	}
+	return &genesis, nil
```

Comment:
Could error on `genesis.Config == nil` or something similar to ensure the json file wasn't empty or other json garbage.

Reply:
I don't see a clear path to that outcome, but a sanity check is fine.